def assert_fail_message(when_failure_print_message: str):
    raise AssertionError(when_failure_print_message)
